client/components/forms/form-phone-input/test/index.jsx

```diff
@@ -1,53 +1,54 @@
-/* eslint-disable vars-on-top */
-
-require( 'lib/react-test-env-setup' )();
-
 /**
  * External dependencies
  */
-var ReactDom = require( 'react-dom' ),
-	React = require( 'react' ),
-	ReactInjection = require( 'react/lib/ReactInjection' ),
-	TestUtils = require( 'react-addons-test-utils' ),
-	expect = require( 'chai' ).expect;
-
-/**
- * Internal dependencies
- */
-var i18n = require( 'lib/mixins/i18n' ),
-	mockCountriesList = require( './mock-countries-list' ),
-	mockCountriesListEmpty = require( './mock-countries-list-empty' );
-
-var countries = mockCountriesList.get();
+var expect = require( 'chai' ).expect,
+	useFakeDom = require( 'test/helpers/use-fake-dom' ),
+	useMockery = require( 'test/helpers/use-mockery' );
 
 describe( 'FormPhoneInput', function() {
-	var FormPhoneInput;
+	var React, ReactDom, ReactInjection, TestUtils, i18n, mockCountriesList, mockCountriesListEmpty, FormPhoneInput, countries, container;
+
+	useFakeDom();
+	useMockery();
 
 	before( function() {
+		ReactDom = require( 'react-dom' );
+		React = require( 'react' );
+		ReactInjection = require( 'react/lib/ReactInjection' );
+		TestUtils = require( 'react-addons-test-utils' );
+
+		i18n = require( 'lib/mixins/i18n' );
+		mockCountriesList = require( './mocks/mock-countries-list' );
+		mockCountriesListEmpty = require( './mocks/mock-countries-list-empty' );
+
+		countries = mockCountriesList.get();
+
 		i18n.initialize();
 		ReactInjection.Class.injectMixin( i18n.mixin );
 		FormPhoneInput = require( 'components/forms/form-phone-input' );
+		container = document.createElement( 'div' );
+		document.body.appendChild( container );
```

Comment:
nope, i can change it. Probably want a new `useFakeDomWIthContainer` too since this is all over the place.